src/components/Match/matchPages.jsx

```diff
@@ -45,45 +45,39 @@ const filterMatchPlayers = (players, team = '') =>
     ((team === 'radiant' && isRadiant(player.player_slot)) || (team === 'dire' && !isRadiant(player.player_slot)) || team === '')
   ).sort((a, b) => a.player_slot - b.player_slot);
 
-const MatchPlayersTable = ({
-  match,
-  columns,
-}) => (<Table data={match.players} columns={columns} />);
-
-const MatchPlayersTableSplit = ({
+const TeamTable = ({
   match,
   columns,
+  heading = '',
 }) => (<div>
+  <Heading title={`${strings.general_radiant} ${heading}`} />
```

Comment:
If you want to avoid the trailing space, you could also use an array with .join()